kuma/static/js/auth.js

```diff
@@ -50,6 +50,16 @@
         });
     });
 
+    // post request to logout user
+    var $logoutButton = $('#logout-button');
+    $logoutButton.click( function (e, service) {
+        var url = $logoutButton.data('logout-url');
+        $.ajax(url, {
```

Comment:
I believe that a successful form submit will need the CSRF token in the data, which isn't available in the page content.